### PR TITLE
added some more descriptions to the JSON schema

### PR DESCRIPTION
**Pull request by beniwohli**
not super detailed, but better than nothing :)

### CODE REVIEW

File: docs/spec/context.json [hunk 1]
```diff
@@ -32,6 +34,7 @@
                     "type": ["boolean", "null"]
                 },
                 "status_code": {
+                    "description": "The HTTTP status code of the response.",
```
Comment (roncohen) [inline docs/spec/context.json]: Hyper text test transfer protocol 😎

File: docs/spec/context.json [hunk 2]
```diff
@@ -41,6 +44,7 @@
         },
         "tags": {
             "type": ["object", "null"],
+            "description": "A mapping of tags with values.",
```
Comment (roncohen) [inline docs/spec/context.json]: flat mapping? to further distinguish itself from custom

**Comment by roncohen** [comment]
nice @beniwohli. We should also make sure these end up on fields.yml

File: docs/spec/context.json (hunk 1 above)
Comment (beniwohli) [reply docs/spec/context.json]: good catch 👍

**Comment by beniwohli** [comment]
@roncohen I suppose there's no automated way to get these into `fields.yml`? 😁

**Comment by roncohen** [comment]
No, but we don't have to do it all at once :)

**Comment by simitt** [comment]
https://github.com/elastic/apm-server/issues/397

**Pull request merged**